shooter_server_2022:
  absolute_wheel_speed: 0.0 # since speed is dynamic now based on distance this is just for debugging and quickly chaning the speeds
  absolute_hood_wheel_speed: 0.0
  eject_speed: 120.0
  error_margin: 7.0
  shooter_wheel_checks: 8
  
